.github/workflows/ci.yml

```diff
@@ -41,20 +41,20 @@ jobs:
     - name: Install vcpkg ports
       shell: bash
       run: |
-        # TinyXML is not installed as a workaround for https://github.com/robotology/ycm/pull/296
-        C:/robotology/vcpkg/vcpkg.exe --overlay-ports=C:/robotology/robotology-vcpkg-ports --overlay-ports=${GITHUB_WORKSPACE}/custom-ports install --triplet x64-windows ace asio boost-asio boost-bind boost-process boost-dll boost-filesystem boost-system freeglut gsl eigen3 esdcan-binary glew glfw3 ode openssl libxml2 libjpeg-turbo eigen3 opencv portaudio matio sdl1 sdl2 qt5-base[latest] ipopt-binary qt5-declarative qt5-multimedia qt5-quickcontrols qt5-quickcontrols2 sqlite3[core,tool]
+        C:/robotology/vcpkg/vcpkg.exe --overlay-ports=C:/robotology/robotology-vcpkg-ports --overlay-ports=${GITHUB_WORKSPACE}/custom-ports install --triplet x64-windows ace eigen3 gsl ipopt-binary libjpeg-turbo opencv portaudio sdl1 sdl2 qt5-base[latest] qt5-declarative qt5-multimedia qt5-quickcontrols qt5-quickcontrols2 sqlite3[core,tool] tinyxml
```

Comment:
I think that `ipopt-binary` probably can be moved to the general archive, I don't think it  is used in YARP.

Reply:
Friendly ping @drdanz , if we prefer we can go forward even by keeping ipopt-binary in the YARP dependencies archive.

Reply:
Sorry, I missed this... You are right, YARP does not use ipopt, I'll move it to the main package